### PR TITLE
Address some minor pylint concerns

### PR DESCRIPTION
**Pull request by keckler**
## Description

This just fixes some pylint concerns that were in a test module that I wrote a while back.

---

## Checklist

- [x] This PR has only one purpose or idea.
- [x] Tests have been added/updated to verify that the new/changed code works.

- [x] The code style follows [good practices](https://terrapower.github.io/armi/developer/standards_and_practices.html).
- [x] The commit message(s) follow [good practices](https://terrapower.github.io/armi/developer/tooling.html).

- [x] The [release notes](https://terrapower.github.io/armi/release/index.html) (location `doc/release/0.X.rst`) are up-to-date with any bug fixes or new features.
- [x] The documentation is still up-to-date in the `doc` folder.
- [x] The dependencies are still up-to-date in `setup.py`.

### CODE REVIEW

File: armi/settings/fwSettings/tests/test_fwSettings.py [hunk 1]
```diff
@@ -141,7 +135,7 @@ def setUp(self):
 
     def test_schema(self):
         # first test that a valid case goes through without error
-        for setting in self.settings.keys():
+        for setting in self.settings:
             # breakpoint()
```
Comment (albeanth) [inline armi/settings/fwSettings/tests/test_fwSettings.py:139]: What's `#breakpoint` for?

**Review by albeanth — COMMENTED** [review]
Would be curious to see how/if this changes with py 3.9 and later versions of pylint 😁

File: armi/settings/fwSettings/tests/test_fwSettings.py (hunk 1 above)
Comment (keckler) [reply armi/settings/fwSettings/tests/test_fwSettings.py]: Whoops, fixed.

**Review by john-science — APPROVED** [review]
Cutting out the cruft and making the code more efficient!

It's small, but I love it anyway.

**Comment by john-science** [comment]
Sorry, I jumped on board this PR late.

We are going to say, for the purpose of `pylint`, that we are testing using Python 3.9 and Pylint 2.14.2.

However, let's take a grain of salt there because those are OBVIOUSLY not standard.  The only way to standardize would be to add a `pylint` CI build to the repo.  But I have looked into it, and we would need to fix a TON of pylint concerns in ARMI first, and _heavily_ narrow and constrain our current `pylintrc` file.  So, we aren't doing that today.  

Having said that...

The changes here amount to removing unused code and improving loops.  Pylint or no, these are obviously good things.

**Comment by keckler** [comment]
Yeah, I'm gonna upgrade to 3.9 and redo, so hold off merging please!

**Comment by keckler** [comment]
@albeanth Okay, I pushed up some more changes. The code is clean as a whistle now!

**Review by albeanth — APPROVED** [review]

**Pull request merged**